docs/migrating-from-0.5.md

```diff
@@ -2,7 +2,122 @@
 
 Genkit 0.9 introduces a number of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.5, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
 
-## 1. CLI Changes
+## Quickstart guide
+
+The following steps will get you up and running on Genkit 0.9 quickly. Read more information about these changes in the detailed [Changelog](#changelog) below.
+
+### 1. Install the new CLI
+
+* Uninstall the old CLI
+
+```posix-terminal
+npm uninstall -g genkit
+```
+
+* Install the new CLI
+
+```posix-terminal
+# Install globally
+npm i -g genkit-cli
+
+# Or install as a dev dependency
+npm i -D genkit-cli
```

Comment:
I'd prefer just one here, or if there are two then why I'd choose one over the other.

Reply:
Pavel had asked me to add in the option to install locally. I'm not actually sure why one would choose one over the other because I'm not a node developer. Are you cool with leaving it for now?

Reply:
I had the same feel as Sam.. prefer to just offer one or give additional guidance on why you might install one vs the other.

Reply:
@pavelgj  what do you think about having this be the local version by default?  I feel like without a `genkit init` there's less of a reason you'd want a global genkit CLI installted

Reply:
Let's tackle this in a followup. I'll leave just the one for now to install locally so that it matches what we have in the get started.